.github/workflows/cleanup_ec2_runners.yml

```
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2014-present, The osquery authors
+#
+# This source code is licensed as defined by the LICENSE file found in the
+# root directory of this source tree.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+
+
+# Cleanup any lingering ec2 runners
+name: cleanup_ec2_runners
+
+on:
+    schedule:
+     - cron: "0 */1 * * *"
+    workflow_dispatch:
+
+jobs:
+    cleanup:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Configure AWS credentials
+              uses: aws-actions/configure-aws-credentials@v3
+              with:
+                aws-access-key-id: ${{ secrets.EC2_GITHUB_RUNNER_AWS_ACCESS_KEY_ID }}
+                aws-secret-access-key: ${{ secrets.EC2_GITHUB_RUNNER_AWS_SECRET_ACCESS_KEY }}
+                aws-region: ${{ secrets.EC2_GITHUB_RUNNER_AWS_REGION }}
+
+            - name: Find Instances
+              run: |
+                cutoff=$(gdate -u --date="-2 hours" "+%Y-%m-%dT%H:%M")
```

Shouldn't it be `date`?

---

Oh yes. On my mac I need `gdate` to get the gnu version, but I forgot to update it.
```suggestion
                cutoff=$(date -u --date="-2 hours" "+%Y-%m-%dT%H:%M")
```